pkg/web/src/app/queries/fetchHelpers.ts

```diff
@@ -30,7 +30,7 @@ export const authorizedFetch = async <TResponse, TData = unknown>(
   const { history, checkExpiry } = fetchContext;
 
   if (ENV.AUTH_REQUIRED !== 'false') {
-    extraHeaders['Authorization'] = `Bearer ${fetchContext.currentUser.access_token}`;
+    extraHeaders['Authorization'] = `Bearer ${fetchContext.currentUser?.access_token}`;
```

Comment:
should we throw an error if `fetchContext.currentUser` is null or missing access_token ?

Reply:
Good question. I don't think it should be possible to hit that case here since routes which need to make authorized requests are also `isProtected`, and if `currentUser` is null the UI will kick the user to the login screen when they try to reach those pages. But in the case where a mistake is made and this is called with a null user / missing token, the request will go through with `Authorization: Bearer [blank]` and then fail with a 401, which will be reported in the component as any other API error would.

Maybe instead we should just not try to add that header at all if the token isn't defined?

Reply:
Pushed a commit to that effect.